BusinessTripAdministration/ViewModels/MainViewModel.cs

```diff
@@ -27,6 +29,8 @@ public string Email
         }
         public MainViewModel()
         {
+            //Thread.Sleep(12000);
```

Comment:
why did you leave the commented code?

Reply:
Csaba used it for a short test and forgot to delete it.